modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -1,119 +1,80 @@
 // Module included in the following assemblies:
-// * installing/installing_aws/installing-aws-localzone.adoc
-// * post_installation_configuration/aws-compute-edge-tasks.adoc
+// * installing/installing_aws/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing_aws/installing-aws-wavelength.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones)
+// * post_installation_configuration/aws-compute-edge-tasks-zones.adoc
 
-ifeval::["{context}" == "aws-compute-edge-tasks"]
-:edge:
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+ifeval::["{context}" == "aws-compute-edge-tasks-zones"]
+:post-aws-zones:
 endif::[]
 
 :_mod-docs-content-type: CONCEPT
 [id="edge-machine-pools-aws-local-zones_{context}"]
-= Edge compute pools and AWS Local Zones
+= Edge compute pools and AWS zones
 
-Edge worker nodes are tainted worker nodes that run in AWS Local Zones locations.
+Edge worker nodes are tainted worker nodes that run in AWS {zone-type} locations.
 
-When deploying a cluster that uses Local Zones, consider the following points:
+When deploying a cluster that uses {zone-type}, consider the following points:
 
-* Amazon EC2 instances in the Local Zones are more expensive than Amazon EC2 instances in the Availability Zones.
-* Latency between applications and end users is lower in Local Zones, and latency might vary by location. A latency impact exists for some workloads if, for example, ingress traffic is mixed between Local Zones and Availability Zones.
+* Amazon EC2 instances in the {zone-type} are more expensive than Amazon EC2 instances in the Availability Zones.
+* Latency between applications and end users is lower in {zone-type}, and latency might vary by location. A latency impact exists for some workloads if, for example, ingress traffic is mixed between {zone-type} and Availability Zones.
 
 [IMPORTANT]
 ====
-Generally, the maximum transmission unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. For more information, see link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation.
-The cluster network MTU must be always less than the EC2 MTU to account for the overhead. The specific overhead is determined by the network plugin, for example:
+Generally, the maximum transmission unit (MTU) between an Amazon EC2 instance in a {zone-type} and an Amazon EC2 instance in the Region is 1300. The cluster network MTU must be always less than the EC2 MTU to account for the overhead. The specific overhead is determined by the network plugin, for example:
 
 - OVN-Kubernetes: `100 bytes`
 - OpenShift SDN: `50 bytes`
```

Comment:
@makentenza can we remove the `OpenShift SDN` from references in the new docs for WL and/or LZ? 4.15+

Reply:
cc @dfitzmau

Reply:
Thanks, Marco. I'll tie in with Marcos on this when the rest of the content is looking good. Marcos agreed to take a look when we get to this stage.

Reply:
Looks like OCP SDN will be removed in 4.15, so I should remove it?

See Jianli's comment on https://issues.redhat.com/browse/OSDOCS-8487

Reply:
@mtulio @dfitzmau, yes sorry, we have been discussing the topic these days as there were some blockers for other architectures but I can confirm you can remove it now from 4.15+